cloudify_agent/installer/config/configuration.py

```diff
@@ -164,7 +164,6 @@ def _cfy_agent_attributes_no_defaults(cloudify_agent):
             cloudify_utils.get_manager_file_server_host()
 
     if not cloudify_agent.get('file_server_port'):
-        # by default, file_server_port is set by an environment variable
```

Comment:
simple because it's true for every other var as well, and pretty clear from the code as it is